### DISCUSSION TITLE
Specify location within workspace on autostart

### DISCUSSION THREAD

**Discussion by ryedunn** [i3/i3#4787] [Q&A] [answered]
I can get two different applications to open vertically within a workspace, that's no problem.

What I'm having difficulties with is getting my browser to open on the right side of the workspace.  I tried changing the order of the exec as well as the assignment with no luck.

How can this be done?

**Comment by Airblader** [comment, marked as answer]
https://i3wm.org/docs/layout-saving.html

**Reply by ryedunn** [reply]
Thank you for the link it was very helpful and I was able to (somewhat) achieve the results what I wanted..

I say somewhat because I can do this with other applications but not with Double Commander.   To clarify, I can put a browser on the right side of the screen and doublecmd on the left (50/50 vsplit), but trying to take it one step further and adding 65% ratio to the browser side, thats when things dont work properly, it will only do an even vsplit. (again, this is **only** when using doublecmd).  I have tested with other applications and it seems to work fine.

To verify, I run workspace-1.json from a terminal and I see the workspace properly split 65% / 35% with the correct titles and stopwatches.   I then run each application and regardless of the order launched, when doublecmd is started, it immediately goes back to a 50/50 split.

Have you heard of this before, perhaps Im doing something wrong, or could it be that DC has its own split window?  I have attached my JSON file if you wanted to look.
[workspace-1.json.txt](https://github.com/i3/i3/files/7857555/workspace-1.json.txt)

Thank you for everything.

EDIT:  Just found out thunar 1.17 supports split windows so I installed it and as anticipated the workspace splits properly.  Although Im still a bit curious about DC, maybe it will help prevent someone else from having the same headache in the future.

**Reply by Airblader** [reply]
If I had to guess, DC probably does some odd things. But I don't have or use it, nor do I feel like installing it, sorry. :-) A log file could be used to take a look at what's happening.

**Reply by ryedunn** [reply]
At least youre honest.  I hope everyone that uses the product feels the same way.
